BFSDFS/나_잡아봐라.py

```
@@ -0,0 +1,41 @@
+
+def catch_me(cony_loc, brown_loc):
+    time = 0
+    queue = deque()
+    queue.append((brown_loc,0))
+    visited = [{} for _ in range(200001)]
+
+    while cony_loc< 200000:
+        cony_loc += time
+        if time in visited[cony_loc]:
+            return time
+        for i in range(0, len(queue)):
```

Ruff (0.13.3)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)